### ISSUE TITLE
[TextArea] Long lines trigger horizontal scrolling, but text should wrap to the next line instead

### ISSUE THREAD

**Issue by nkohari** [auth0/cosmos#243]
Here's an example from the Advanced tab on the Clients screen in the PoC:

![screen shot 2018-02-08 at 12 16 38 pm](https://user-images.githubusercontent.com/1576/35987615-2b3359aa-0cca-11e8-99ab-7df292ef8463.png)

**Comment by siddharthkp** [comment]
@landitus Broke in https://github.com/auth0/cosmos/commit/8cf653d89a7c55d9967700aa2e0ffbaf96976c65

Not sure what the `white-space: pre` is for 🤔. Can we remove it?

**Comment by landitus** [comment]
jajaj, yes, I guess I broke it. I'm starting to figure out how the white-space works, I think. The white-space pre depends on how the text is formatted (if it has line breaks or not). I've changed the value with a continuous string (no line breaks) and now it looks bad. We'll need to check how the content will be set by our users. And document it.

For now, we can remove `white-space:pre` to make it look better.
